ff-asn-apply-filters/index.html

```diff
@@ -0,0 +1,245 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>ASN "Apply Filters" Demo</title>
+  <link rel="stylesheet" href="../assets/demo-styles.css">
+
+  <script src="../node_modules/ff-web-components/dist/vendor/custom-elements-es5-adapter.js"></script>
+  <script src="../node_modules/ff-web-components/dist/vendor/webcomponents-loader.js"></script>
+  <script defer src="../node_modules/ff-web-components/dist/bundle.js"></script>
+
+  <style>
+    main {
+      display: flex;
+      padding: 0 10px;
+    }
+
+    .filters {
+      flex: 0 0;
+      width: 240px;
+      margin-right: 10px;
+    }
+
+    .products-listing {
+      flex: 1 0;
+    }
+
+    ff-record-list {
+      margin-right: -10px;
+    }
+  </style>
+
+  <script>
+    const applyFilters = (group, button, eventAggregator) => () => {
+      const selected = Array.from(group.querySelectorAll('ff-asn-group-element') || [])
+        .filter(g => g.selected)
+        .map(g => g.element)
+
+      if (selected) {
+        eventAggregator.addFFEvent({
+          type: 'search',
+          filter: selected.map(e => `${e.associatedFieldName}:${e.name}`),
+          success: () => {
+            button.classList.remove('btn--pending')
+            button.disabled = true
+          }
+        })
+      }
+    }
+
+    document.addEventListener('ffReady', function (event) {
+      const eventAggregator = event.eventAggregator;
```

Comment:
There is some inconsistency in semicolon usage. Could we use semicolons everywhere to be consistent with other demos?